copier.yml

```diff
@@ -76,6 +76,15 @@ ci:
     - GitHub
   help: Which CI system to use ?
 
+odoo_test_flavor:
+  type: str
+  default: Both
```

Comment:
Do we want to set the default value to OCB for Odoo < 13 ?

Reply:
We already have `default: "https://github.com/{{ org_slug }}/{{ repo_slug }}"` so we probably can put any python expression in there?

Reply:
Yes, you can do that.